app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -230,7 +230,9 @@ key --skip
 <% end -%>
 
 text
-reboot
+reboot <% if host_param_true?('install_reboot_kexec') -%>
```

Comment:
```
-reboot
+reboot 
```

Lol, now it's playing with us. The offender is the space after reboot.

How about
```
reboot<% if ... %> --kexec<% endif %>
```

Reply:
Swapped to that, hopefully that makes it happier :)